test/runner/test.py

```diff
@@ -192,7 +192,7 @@ def parse_args():
                       help=argparse.SUPPRESS)
 
     add_changes(test, argparse)
-    add_environments(test)
+    add_environments(test, None)
```

Comment:
This change doesn't appear to be needed.

Reply:
Thanks, accidentally left it there when testing a few things, have fixed it.